setup.py

```diff
@@ -22,7 +22,7 @@ def read(*filenames, **kwargs):
 
 setup(
     name='pyrk',
-    version=get_git_version(),
+    #version=get_git_version(),
```

Comment:
ok, cool. we can leave it commented. I'll figure out what's happening when you're building it.